subsys/jaxrs/src/main/java/org/commonjava/indy/bind/jaxrs/IndyDeployment.java

```diff
@@ -174,7 +175,7 @@ public DeploymentInfo getDeployment( final String contextRoot )
                                                                             DispatcherType.REQUEST )
                                                       .setDeploymentName( "Indy" )
                                                       .setClassLoader( ClassLoader.getSystemClassLoader() )
-                                                      .addOuterHandlerChainWrapper( new HeaderDebugger.Wrapper() );
+                                                      .addOuterHandlerChainWrapper( new HeaderDebugger().new Wrapper() );
```

Comment:
I think we should just comment out this line, and maybe add a comment to the effect that this wrapper is suspected of performing poorly, and we should be careful not to leave it in effect in production deployments.

Reply:
I agree also.